scripts/build-plugin-list.js

```diff
@@ -48,7 +48,7 @@ async function extractEcosystemFromFile(file) {
 
 function extractPlugins(pluginContent) {
   const lines = pluginContent.split('\n').filter(Boolean) // remove empty lines
-
+  console.log(lines)
```

Comment:
could you remove the changes here?

so this PR can focus on 1 single thing at time

Reply:
done